keps/sig-api-machinery/1929-built-in-default/README.md

```diff
@@ -398,12 +402,10 @@ which would have the following behavior, both for CRD types and built-in types:
 {"entry": {"name": "other-name", "number": 0}}
 ```
 
-#### Non-pointer fields
+#### Non-pointer scalar fields
 
-Non-pointer fields are also important, since they have a specific unmarshalling mechanism in Go.
-
-For CRDs, we can assume that invalid zero-values are omitted, otherwise
-the objects would technically be invalid.
+Non-pointer scalar fields also have a specific unmarshalling mechanism in Go,
+since they always have a value.
 
 For built-ins, we can differentiate two cases:
 1. The zero-value for the field is invalid (e.g. if empty string is not a valid value)
```

Comment:
I don't see an example for this case. What does this even mean to be "invalid" ?